modules/benchmark/R/calc.benchmark.R

```diff
@@ -40,29 +46,83 @@ calc.benchmark <- function(bm.ensemble, con){
     bm.ids <- bms$id[which(bms$input_id == obvs.id)]
     data.path <- query.file.path(obvs.id,wf$hostname,con)
     format_full <- format <- query.format.vars(obvs.id,con) 
-    # format is a list, metrics is a dataframe, both can be long. How to pass to calc.metrics remotely? 
-    # Maybe as an Rdata file?
+
+    # ---- LOAD INPUT DATA ---- #
+    # Right now I'm making the inappropriate assumption that storage type will be 
+    # empty unless it's a time variable. 
+    # This is because I haven't come up for a good way to test that a character string is a date format
+
+    format$vars <- format_full$vars %>% filter(., variable_id %in% db.query(paste(
+      "SELECT variable_id from benchmarks WHERE id IN (", paste(bm.ids, collapse = ","),");"),con)[[1]] | 
+        nchar(storage_type) > 0)  
+    st <- format$vars$storage_type
+    time.row <- which(nchar(st)>1 & substr(st, 1,1) == "%")
```

Comment:
I thought we'd discussed explicitly flagging the time row in the format record?  Seems much easier than trying to infer it indirectly. Fine for now, but lets sort this out for v1.4.9